app/poetry.lock

```diff
@@ -1585,11 +1585,11 @@ test = ["pytest", "pytest-cov"]
 
 [[package]]
 name = "xlsxwriter"
-version = "1.4.5"
+version = "3.0.2"
```

Comment:
That's a big version boost. Should be fine, generally in favour but a candidate if there's any problems... with writing Excel spreadsheets, which I don't think anything does?